packages/modules/manage-versions/src/components/InformationPanel/ChangesetInformationPanel.scss

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+.iac-info-panel {
+  width: 400px;
+  min-width: 400px;
+  .iac-info-panel-body {
+    padding: var(--iui-size-m);
+    .iac-info-panel-container {
+      border-top: 1px solid var(--iui-color-background-backdrop);
```

Comment:
I thiknk there is border specific variable.. no?

Reply:
Fixed. Using var(--iui-color-border-subtle);